v1/metrics.yaml

```diff
@@ -92,10 +92,22 @@ paths:
           description: Error
           schema:
             $ref: '#/definitions/problem'
+      x-route-filters:
+        - type: default
+          name: ratelimit_route
+          options:
+            limits:
+              internal: 100
+              external: 100
+            log_only: true
       x-request-handler:
         - get_from_backend:
             request:
               uri: '{{options.host}}/pageviews/per-article/{project}/{access}/{agent}/{article}/{granularity}/{start}/{end}'
+              headers:
+                cache-control: '{{cache-control}}'
```

Comment:
This is unlikely to do much for external requests, as Varnish strips cache-control headers.